labelbox/data/serialization/ndjson/classification.py

```diff
@@ -100,17 +122,29 @@ class NDRadioSubclass(NDFeature):
     answer: NDFeature
 
     def to_common(self) -> Radio:
-        return Radio(
-            answer=ClassificationAnswer(name=self.answer.name,
-                                        feature_schema_id=self.answer.schema_id,
-                                        confidence=self.answer.confidence))
+        classifications = [
```

Comment:
Can we put this inside `Radio` to keep it consistent with how it's done in `Checklist`?
```
return Radio(answer=ClassificationAnswer(
            name=self.answer.name,
            feature_schema_id=self.answer.schema_id,
            confidence=self.answer.confidence,
            classifications=[
                NDSubclassification.to_common(annot)
                for annot in self.answer.classifications
        ],
        ))
```

Reply:
done